openshift/olm/knative-serving.v0.4.1.clusterserviceversion.yaml

```diff
@@ -256,6 +256,7 @@ spec:
           - config.openshift.io
           resources:
           - networks
+          - ingresses
```

Comment:
also, it's strange that these diffs are different. it appears you have the order of the apiGroups stanzas backwards. i'm not sure if order matters, but it's weird to have them different.

Reply:
Not sure I get this.   I added this line so it shows up as diff.   What am I missing?